RandomizerHost/ViewModels/MainWindowViewModel.cs

```diff
@@ -167,7 +177,22 @@ public async void CreateFromRandomSeed(Window in_Window)
         {
             try
             {
-                this.PerformRandomization(-1);
+                // Create a random string of numbers to use as the seed
```

Comment:
I think it's better to do this by creating a random `UInt32` (allowing the full range) and then using the existing stuff to convert that to base26. The way this code works it always generates a 10 digit number as a hash which isn't going to give the full range. Plus the other way would be less code. It's the way I do it in Rollchan.

Reply:
Or if we go with the `Seed` type, you could just create a new one from the entropy source and call `ToBase26String()`.

Reply:
Yeah, I suppose we could do that, I just wanted it to be a constant size.